cmd/fission-bundle/main.go

```diff
@@ -296,7 +294,16 @@ Options:
 	if arguments["--storageServicePort"] != nil {
 		port := getPort(logger, arguments["--storageServicePort"])
 		filePath := arguments["--filePath"].(string)
-		runStorageSvc(logger, port, filePath)
+
+		if arguments["--storageType"] != nil && arguments["--storageType"] == storagesvc.StorageTypeS3 {
+			bucketName := os.Getenv("STORAGE_S3_BUCKET_NAME")
+			accessKeyId := os.Getenv("STORAGE_S3_ACCESS_KEY_ID")
+			secretAccessKey := os.Getenv("STORAGE_S3_SECRET_ACCESS_KEY")
+			region := os.Getenv("STORAGE_S3_REGION")
+			runStorageSvc(logger, port, filePath, "s3", bucketName, accessKeyId, secretAccessKey, region)
```

Comment:
Can this go inside the storageSvc - as this is impl detail of Storage Service.

Reply:
Problem with that is my little knowledge of golang. As far as I know we can't have same functions with different signatures. Therefore one single function `runStorageSvc` won't be able to handle for both `pvc/local` and `s3` cases. 

Which is why I feel we need to bisect it here and call with appropriate function signature. I will give this a thought again.

Reply:
I've put this inside the `NewS3Storage` function. There we will read these env variables and create a storage stuct out of it. This is also better for case when we want to move to file based config ( from env based config). That way we will read from config file in same place. Views ?